PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -236,9 +284,12 @@ struct HfCandidateCreatorXic0Omegac0 {
     registry.add("hKFParticleDcaXYCascBachToPv", "hKFParticleDcaXYCascBachToPv", kTH1F, {{1000, -0.1f, 100.0f}});
     registry.add("hKfLambda_ldl", "hKfLambda_ldl", kTH1F, {{1000, 0.0f, 1000.0f}});
     registry.add("hKfOmega_ldl", "hKfOmega_ldl", kTH1F, {{1000, 0.0f, 1000.0f}});
+    registry.add("hKfXi_ldl", "hKfXi_ldl", kTH1F, {{1000, 0.0f, 1000.0f}});
```

Comment:
change all `TH1F` to `TH1D`, even those pre-existing in the code. With Run3 statistics, with `TH1F` one risks to saturate with the floating-point precision (pow(2,24)-1 = 16777215).